internal/testutil/tempdir.go

```diff
@@ -34,20 +34,19 @@ type TempDir struct {
 
 // NewTempDir creates a temporary directory and a cleanup function.
 // It is the responsibility of calling code to call cleanup when done.
```

Comment:
This godoc should be updated.

Very cool though!

Reply:
Done. Thanks for spotting!